lib/components/AnimatedRegion.ts

```diff
@@ -26,7 +29,7 @@ const defaultValues = {
 
 let _uniqueId = 1;
 
-const getAnimatedValue = (valueIn, fallback) => {
+const getAnimatedValue = (valueIn: unknown, fallback: number) => {
```

Comment:
Technically, this isn't `unknown`. This would be more

```js
type TValueIn = number | Animated.Value;

const getAnimatedValue = (valueIn?: TValueIn, fallback: number) => { ... };
```

This is why we conditionally use the `fallback`.

Reply:
You are correct. I didn't make it optional however, so that we can preserve the order of the parameters.